src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py

```diff
@@ -59,34 +59,35 @@
 				Label:
 					id: title_label
 					size_hint: (None,None)
-					height: dp(60)
-					width: dp(800)
+					height: dp(0.125*app.height)
+					width: dp(1.0*app.width)
 					text: "SmartBench Warranty Registration"
 					color: hex('#f9f9f9ff')
 					# color: hex('#333333ff') #grey
-					font_size: dp(30)
+					font_size: dp(0.0375*app.width)
 					halign: "center"
 					valign: "bottom"
 					markup: True
 				   
 			# BODY
 			BoxLayout:
 				size_hint: (None,None)
-				width: dp(800)
-				height: dp(298)
+				width: dp(1.0*app.width)
+				height: dp(0.620833333333*app.height)
 				orientation: 'vertical'
 
 
 				Button:
+				    font_size: str(0.01875 * app.width) + 'sp'
```

Comment:
check indentation

Reply:
This is just github being weird and displaying tabs/spaces differently, it looks correct in the editor and works properly

Reply:
![image](https://github.com/YetiTool/easycut-smartbench/assets/75572055/c917f141-3bd7-44d6-98a9-83882a62f3f3)